From e6277e64e505bbb6872cfc4b8945641f08332f21 Mon Sep 17 00:00:00 2001
From: Yang Fan F <fan.f.yang@intel.com>
Date: Wed, 29 Apr 2015 01:58:42 +0800
Subject: [PATCH 07/28] Revert "atomisp2: Fix hmm memtrack report"

This reverts commit c9554e1a9a416ecc13f0a63a240d5f0de342db24.

Change-Id: I4f5f39fbf6ec01d24a7f2f9badedd9d346fa7c8d
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9279
Signed-off-by: Yang Fan F <fan.f.yang@intel.com>
Reviewed-on: https://android.intel.com:443/440493
---
 .../camera/drivers/media/pci/atomisp2/atomisp_driver/hmm/hmm.c        | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/hmm/hmm.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/hmm/hmm.c
index 358d340..f61dd6c 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/hmm/hmm.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/hmm/hmm.c
@@ -78,8 +78,8 @@ static ssize_t bo_show(struct device *dev, struct device_attribute *attr,
 
 	spin_lock_irqsave(&bo_device.list_lock, flags);
 	list_for_each_entry(bo, bo_list, list) {
-		if ((active && (bo->status & HMM_BO_ALLOCED)) ||
-			(!active && !(bo->status & HMM_BO_ALLOCED))) {
+		if ((active && !(bo->status & HMM_BO_FREE)) ||
+			(!active && (bo->status & HMM_BO_FREE))) {
 			ret = scnprintf(buf + index1, PAGE_SIZE - index1,
 				"%s %d\n",
 				hmm_bo_type_strings[bo->type], bo->pgnr);
-- 
1.9.1

